landing_page/templates/landing_page/base.html

```diff
@@ -52,8 +54,8 @@ <h1>ReeldIn</h1>
   </header>
 
   <nav>
-    <a href="{% url 'landing_page:index' %}">Home</a>
-    <a href="{% url 'landing_page:about' %}">About</a>
+    <a href="index.html">Home</a>
+    <a href="about.html">About</a>
```

Comment:
These links need to be the previously deleted ones 

<a href="{% url 'landing_page:index' %}">Home</a>
 <a href="{% url 'landing_page:about' %}">About</a>
 
 regular references won't work in our django environment.